x-hooks/api/marketplace/management/allow-list/use-delete-allow-list-entry.ts

```diff
@@ -1,6 +1,19 @@
-import {api} from "../../../../../services/api";
+import { AllowListTypes } from "interfaces/enums/marketplace";
 
-export default async function useDeleteAllowListEntry(networkId: number, address: string) {
-  return api.delete<string[]>(`/marketplace/management/${networkId}/whitelist/${address}`)
+import { api } from "services/api";
+
+interface useDeleteAllowListEntry {
+  networkId: number;
+  address: string;
+  type: AllowListTypes;
+}
+
+export default async function useDeleteAllowListEntry ({
```

Comment:
objects have a whole lot that can go wrong, return to use primitives as arguments (aka `networkId: number, address: string, type: "add"|"remove"` (with the added bonus that you don't need to write a _type_ just for these arguments)